content/hi/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,649 @@
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}},
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}, और
+{{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}.
+
+<!-- body -->
+
+## प्रबंध
+
+{{< glosary_tooltip text="API server" term_id="kube-apiserver" >}} में नोड्स जोड़ने के दो मुख्य तरीके हैं:
+
+1. नोड पर क्यूबलेट नियंत्रण विमान में स्व-पंजीकृत होता है
+2. आप (या अन्य मानव उपयोगकर्ता) मैन्युअल रूप से नोड ऑब्जेक्ट जोड़ते हैं
+
+आपके द्वारा एक नोड बनाने के बाद {{< glosary_tooltip text="object" term_id="object" >}},
+या नोड पर क्यूबलेट स्व-पंजीकरण करता है, नियंत्रण विमान यह जांचता है कि नया नोड ऑब्जेक्ट है या नहीं
+वैध। उदाहरण के लिए, यदि आप निम्न JSON मेनिफेस्ट से नोड बनाने का प्रयास करते हैं:
+
+```json
+{
+  "kind": "Node",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "10.240.79.157",
+    "labels": {
+      "name": "my-first-k8s-node"
+    }
+  }
+}
+```
+
+कुबेरनेट्स आंतरिक रूप से एक नोड वस्तु बनाता है (प्रतिनिधित्व)। कुबेरनेट्स जाँच करता है
+कि एक क्यूबलेट एपीआई सर्वर पर पंजीकृत है जो `metadata.name` से मेल खाता है
+नोड का क्षेत्र। यदि नोड स्वस्थ है (यानी सभी आवश्यक सेवाएं चल रही हैं),
+तो यह पॉड चलाने के योग्य है। अन्यथा, उस नोड को किसी भी क्लस्टर गतिविधि के लिए अनदेखा कर दिया जाता है
+जब तक यह स्वस्थ न हो जाए।
+
+{{< नोट >}}
+कुबेरनेट्स ऑब्जेक्ट को अमान्य नोड के लिए रखता है और यह देखने के लिए जांच जारी रखता है कि क्या
+यह स्वस्थ हो जाता है।
+
+आपको, या किसी {{< glosary_tooltip term_id="controller" text="controller">}} को स्पष्ट रूप से
+उस स्वास्थ्य जाँच को रोकने के लिए नोड वस्तु को हटा दें।
+{{< /नोट >}}
+
+नोड ऑब्जेक्ट का नाम मान्य होना चाहिए
+[DNS सबडोमेन नाम](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)।
+
+### नोड नाम विशिष्टता
+
+[नाम](/docs/concepts/overview/working-with-objects/names#names) एक नोड की पहचान करता है। दो नोड्स
+एक ही समय में एक ही नाम नहीं हो सकता। कुबेरनेट्स भी मानते हैं कि उसी के साथ एक संसाधन
+नाम एक ही वस्तु है। नोड के मामले में, यह स्पष्ट रूप से माना जाता है कि एक उदाहरण का उपयोग कर रहा है
+समान नाम की स्थिति समान होगी (उदा. नेटवर्क सेटिंग्स, रूट डिस्क सामग्री)
+और विशेषताएँ जैसे नोड लेबल। इससे हो सकता है
+विसंगतियां अगर किसी उदाहरण को उसका नाम बदले बिना संशोधित किया गया था। यदि नोड की आवश्यकता है
+प्रतिस्थापित या महत्वपूर्ण रूप से अपडेट किया गया, मौजूदा नोड ऑब्जेक्ट को एपीआई सर्वर से हटाने की जरूरत है
+अपडेट के बाद पहले और फिर से जोड़ा गया।
+
+### नोड्स का स्व-पंजीकरण
+
+जब क्यूबलेट फ्लैग `--register-node` ट्रू (डिफ़ॉल्ट) होता है, तो क्यूबलेट प्रयास करेगा
+एपीआई सर्वर के साथ खुद को पंजीकृत करें। यह पसंदीदा पैटर्न है, जिसका उपयोग अधिकांश डिस्ट्रोज़ द्वारा किया जाता है।
+
+स्व-पंजीकरण के लिए, क्यूबलेट निम्नलिखित विकल्पों के साथ शुरू होता है:
+
+- `--kubeconfig` - API सर्वर पर स्वयं को प्रमाणित करने के लिए क्रेडेंशियल्स का पथ।
+- `--cloud-provider` - {{< glosary_tooltip text="क्लाउड प्रोवाइडर" टर्म_आईडी="क्लाउड-प्रोवाइडर" >}} से कैसे बात करें अपने बारे में मेटाडेटा पढ़ने के लिए।
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, the localization of the shortcode in this line looks incorrect.

```suggestion
- `--cloud-provider` - {{< glossary_tooltip text="क्लाउड प्रदाता" term-id="cloud-provider" >}} से कैसे बात करें अपने बारे में मेटाडेटा पढ़ने के लिए।
```